.circleci/config.yml

```diff
@@ -151,9 +187,9 @@ workflows:
   version: 2
   build_all:
     jobs:
+      - build-php73-laravel58
+      - build-php72-laravel58
+      - build-php71-laravel58
       - build-php73-laravel57
```

Comment:
To reduce the amount of builds we can drop `build-php73-laravel57` and `build-php72-laravel57` as well and build Laravel `5.7` also just with the lowest possible version.